docs/diagrams/CopySequenceDiagram.puml

```diff
@@ -0,0 +1,75 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":InternBuddyParser" as InternBuddyParser LOGIC_COLOR
+participant ":CopyCommandParser" as CopyCommandParser LOGIC_COLOR
+participant "v:CopyCommand" as CopyCommand LOGIC_COLOR
+participant "<<class>>\nParserUtil" as ParserUtil LOGIC_COLOR
+participant "cr:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute("copy 1")
+activate LogicManager
+
+LogicManager -> InternBuddyParser : parseCommand("copy 1")
+activate InternBuddyParser
+
+create CopyCommandParser
+InternBuddyParser -> CopyCommandParser
+activate CopyCommandParser
+
+CopyCommandParser --> InternBuddyParser
+deactivate CopyCommandParser
+
+InternBuddyParser -> CopyCommandParser : parse("1")
+activate CopyCommandParser
+
+CopyCommandParser -> ParserUtil : parseIndex("1")
+activate ParserUtil
+
+ParserUtil --> CopyCommandParser : indexObject
+deactivate ParserUtil
+
+create CopyCommand
+CopyCommandParser -> CopyCommand : CopyCommand(indexObject)
+activate CopyCommand
+
+CopyCommand --> CopyCommandParser
+deactivate CopyCommand
+
+CopyCommandParser --> InternBuddyParser : v
```

Comment:
Will it be better to label the `CopyCommand` object as `c`? I labelled `v` for `ViewCommand` because `v` stands for `View`.

Reply:
Thank you for pointing out!